ASSEMBLY.md

```diff
@@ -85,9 +85,11 @@ The Board Management Controller (BMC) is responsible for controlling power and r
 3. Ensure you have the Rust Programming Language installed (see <https://www.rust-lang.org>)
 4. Install the `probe-run` tool:
 
-```console
-$ cargo install probe-run
-```
+	```console
```

Comment:
Unsure what this indentation is for?

Reply:
Elements in lists need to be indented according to https://www.markdownguide.org/basic-syntax/ under "Adding Elements in Lists"